### DISCUSSION TITLE
Is it possible to implement an intro playlist?

### DISCUSSION THREAD

**Discussion by amarildoazura** [AzuraCast/AzuraCast#3880] [Q&A]
**Using Docker installation method**

**AzuraCast version**

v0.11.1 Stable • Docker • PHP 7.4

**Host Operating System** 
UBUNTU 20.04 VPS CONTABO CPU 6 NUCLEOS 16BR RAM 400GB HD
**Describe the bug**
Hello AzuraCast team, first of all I would like to say that I was very happy with the new updates that fixed problems with repeated vignettes and it seems the system has become lighter.

I would like to know how I can make an opening file for a certain schedule and make that audio play before starting the playlist:
Example: at 22:00 hours in São Paulo Brazil this opening audio would indicate the beginning of the program and then the playlist with the songs of that program, in my case it would be a program with Italian songs, this introduction audio has approximately 1 minute and 20 seconds.

By the tests that I did, creating a playlist starting at 22:00 and ending at 22:01, it appeared the countdown but did not play the song from the previous list ended at 22:04 and then the playlist of Italian songs, also test with just 1 minute, following the list as a normal sequential option according to the images.

I wonder if you have how to do this and how.

I thank the attention.

![01](https://user-images.githubusercontent.com/70643154/101838543-94b75580-3b1f-11eb-94da-09926a2c6b94.PNG)
![02](https://user-images.githubusercontent.com/70643154/101838553-9a14a000-3b1f-11eb-8b24-d159cb665ce9.PNG)
![03](https://user-images.githubusercontent.com/70643154/101838557-9d0f9080-3b1f-11eb-8344-66c2ddf41e5b.PNG)
![04](https://user-images.githubusercontent.com/70643154/101838574-a39e0800-3b1f-11eb-87fa-8b1f3b1a0564.PNG)

**To Reproduce**

**Expected behavior**

**Relevant Logs**

**Screenshots**

**Device(s):**
 - Device:  
 - OS: 
 - Browser: 

**Additional context**
 
Would it be possible to implement an intro playlist?

**Comment by amarildoazura** [comment]
Thanks Vaalyn the translation from Brazilian Portuguese to English did not go as it should

**Comment by hyperbrute** [comment]
Yes please, intro and outro for playlist would be great !
thanks

**Comment by Guldstrand** [comment]
Yes, it would have been really great if it was possible to choose both intro and outro for playlists.
Can you please, please try to implement this?

**Comment by Moonbase59** [comment]
There is still interest in this (me, too), you might want to upvote the [feature request](https://features.azuracast.com/suggestions/449500/add-option-within-the-playlist-insert-opening-and-closing-audio-vignette).
